File: HelloWorld/src/main/java/de/fh_aachen/android/unit00/hello_world/MainActivity.java
package de.fh_aachen.android.unit00.hello_world;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClickButton(View v) {
        Toast.makeText(this,"Pressed!",Toast.LENGTH_LONG).show();
    }

}

File: anborn_k/src/main/java/de/fh_aachen/android/exercises/anborn_k/MainActivity.kt
package de.fh_aachen.android.exercises.anborn_k

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClickButton(v: View?) {
        Toast.makeText(this, "Pressed!", Toast.LENGTH_LONG).show()
    }

}
